velox/functions/lib/DateTimeFormatter.h

```diff
@@ -218,10 +218,10 @@ class DateTimeFormatter {
 std::shared_ptr<DateTimeFormatter> buildMysqlDateTimeFormatter(
     const std::string_view& format);
```

Comment:
Should we also make this method return Expected for consistency?

Reply:
Of course. Updated.